spec/feature/posts_index_spec.rb

```diff
@@ -1,36 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe 'User post index page', type: :feature do
-  let!(:user) { User.create(name: 'Sarkodie', photo: '/path/to/photo1.jpg', post_counter: 10) }
+  let!(:user) { User.create(name: 'Sarkodie', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', posts_counter: 10) }
 
   before do
     5.times do
       Post.create(
         title: 'Post title',
         text: 'Post text',
         author_id: user.id,
-        comments_counter: 3,
-        likes_counter: 7
+        comments_counter: 2,
+        likes_counter: 1
       )
     end
     visit user_posts_path(user)
   end
 
-  it 'displays user profile information' do
```

Comment:
- [ ] You are currently missing the following tests:
- Test that asserts you can see the user's profile picture.
- Test that asserts you can see the user's username.
Kindly add these missing test cases